src/test/resources/message-viewer/consume-proto-topic-expected-response.json

```diff
@@ -25,7 +25,7 @@
               "value": "NTIyMjMz"
             }
           ],
-          "key": "522233",
+          "key": 522233,
```

Comment:
Is this because we used to be always converting keys and responses to string representations? I couldn't figure out whether the actual key is a numeric or a string. Can you please confirm?